arch/arm/include/inttypes.h

```diff
@@ -29,91 +29,142 @@
  * Pre-processor Prototypes
  ****************************************************************************/
 
+#define signed      +0
+#define unsigned    +0
+#define int         +2
+#define long        +4
```

Comment:
Just for my understanding, can it be `+1`, `+2`? Or 2 and 4 are some magic numbers?

Reply:
+1 if for `uint16_t` if you want to know `uint16_t` typedef to `short int` or `int` since gcc/clang typedef uint16_t to short, I omit the related code. If you are interesting please reference newlib link I gave above.